sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClient.java

```diff
@@ -812,6 +812,15 @@ Flux<ServiceBusReceivedMessage> peekMessages(int maxMessages, long sequenceNumbe
      *     <li>An {@link AmqpException} occurs that causes the receive link to stop.</li>
      * </ul>
      *
+     * <p>
+     * The client uses an amqp-link underneath to receive the messages; the client will transparently transition to
```

Comment:
nit: amqp-link -> "AMQP link"

"The client uses an amqp-link underneath to receive the messages;" -> "The client uses an AMQP link to receive the messages;

Reply:
thanks, updated